notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        Given an game info dict, returns the number of flops used in that game.
</pre><p>"Given an game"-&gt;"Given a game"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Fixed.